packages/commerce-sdk-react/src/hooks/ShopperCustomers/cache.ts

```diff
@@ -4,9 +4,11 @@
  * SPDX-License-Identifier: BSD-3-Clause
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
+import {ShopperCustomersTypes} from 'commerce-sdk-isomorphic'
 import {Query} from '@tanstack/react-query'
-import {getCustomerProductListItem, QueryKeys} from './queryKeyHelpers'
-import {ApiClients, CacheUpdate, CacheUpdateMatrix, Tail} from '../types'
+import clone from 'clone'
```

Comment:
After gzip `clone` is only [1kb](https://bundlephobia.com/package/clone@2.1.2). The others would be 327 bytes and 410 bytes. I'll throw out `clone-json` because it only has 1 weekly download, `just-clone` has 55k weekly downloads, "clone" has 55m weekly downloads. 
So clone of just-clone. Up for debate, I'll think about that.